### PR TITLE
Fix namespace and improve assert equals

### PR DESCRIPTION
**Pull request by peter279k**
As title.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,13 +44,15 @@
...
-            "Jaeger\\": "src/Jaeger/",
-            "Jaeger\\Tests\\": "tests/Jaeger/"
+            "Jaeger\\": "src/Jaeger/"
         },
         "files": [
             "./src/Jaeger/Constants.php"
         ]
     },
+    "autoload-dev": {
+        "Jaeger\\Tests\\": "tests/Jaeger/"
```
Comment (peter279k) [inline composer.json:54]: This namespace should move to `autoload-dev` block.

File: tests/Jaeger/Codec/ZipkinCodecTest.php [hunk 2]
```diff
@@ -40,7 +40,7 @@ function testInject()
         $this->assertEquals('7b', $carrier['X-B3-TraceId']);
         $this->assertEquals('1c8', $carrier['X-B3-SpanId']);
         $this->assertEquals('315', $carrier['X-B3-ParentSpanId']);
-        $this->assertEquals('1', $carrier['X-B3-Flags']);
+        $this->assertSame(1, $carrier['X-B3-Flags']);
```
Comment (peter279k) [inline tests/Jaeger/Codec/ZipkinCodecTest.php:43]: This `$carrier['X-B3-Flags']` will be `1` and it should use the `assertSame` to assert equals checking strictly.

File: tests/Jaeger/Sampler/RateLimitSamplerTest.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Jaeger\Test\Sampler;
+namespace Jaeger\Tests\Sampler;
```
Comment (peter279k) [inline tests/Jaeger/Sampler/RateLimitSamplerTest.php:3]: This class namespace should be fixed.

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you!

**Pull request merged**